src/docs/release/breaking-changes/android-setIsRunningInRobolectricTest-removed.md

```diff
@@ -0,0 +1,60 @@
+---
+title: FlutterMain.setIsRunningInRobolectricTest on Android removed
+description: The test-only FlutterMain.setIsRunningInRobolectricTest API on
```

Comment:
The description must be one line, so let it wrap. This might be why Travis failed.

Reply:
thanks